webui/react/package.json

```diff
@@ -24,18 +24,21 @@
     "dayjs": "^1.10.4",
     "eventemitter3": "^4.0.7",
     "fp-ts": "^2.8.4",
+    "fuse.js": "^6.4.6",
     "history": "^4.10.1",
     "humanize-duration": "^3.24.0",
     "io-ts": "^2.2.11",
     "js-sha512": "^0.8.0",
     "js-yaml": "^3.14.0",
+    "omnibar": "github:hamidzr/omnibar",
     "plotly.js": "^1.56.0",
     "portable-fetch": "^3.0.0",
     "pretty-bytes": "^5.4.1",
     "query-string": "^6.13.5",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-helmet-async": "^1.0.9",
+    "react-hotkeys": "^2.0.0",
```

Comment:
**question:** thoughts on using the `useKeyTracker` event bus versus including a `react-hotkeys` library? `GalleryModal` has an example on how to use our `useKeyTracker` on the listening side.

`react-hotkeys` is not a big bundle by any means, but for the simple use case of just changing a store bit when a key combo is detected, the `useKeyTracker` should suffice?

<img width="251" alt="Screen Shot 2021-05-12 at 3 27 49 PM" src="https://user-images.githubusercontent.com/220971/118046564-b9199700-b336-11eb-9e78-ccbf1dc5bd5e.png">

Reply:
i converted it and it was pretty nice to use. + fixed the escape issue